hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/StorageContainerManager.java

```diff
@@ -1708,4 +1708,17 @@ ContainerTokenSecretManager getContainerTokenSecretManager() {
     return containerTokenMgr;
   }
 
+  @Override
+  public List<String> getScmRatisRoles() throws IOException {
+    return getScmHAManager().getRatisServer().getRatisRoles();
+  }
+
+  @Override
+  public String getPrimordialNode() {
+    if (SCMHAUtils.isSCMHAEnabled(configuration)) {
+      return SCMHAUtils.getPrimordialSCM(configuration);
+    }
+    return null;
```

Comment:
If config primordial node is not defined, the node which is performed is considered as primordial SCM.
We persist scmID in the version file but not the hostname. So, I don't see a way we can figure out primordial SCM. So, I believe we should be okay here if the config is not defined return null (And print some message that init performed node is considered as primordial SCM)

cc @bshashikant for comments

And also primordial SCM can be nodeID/hostName. If it is nodeID, get the hostName from SCMNodeDetails and use that.

Reply:
If the prmiodial node is not defined, let's not not show it at all.

Reply:
Addressed comment. If primordial node is not defined , the  field will be hidden.